File: TestApp/app/src/main/java/com/example/dana_maria/testapp/MainActivity.java
package com.example.dana_maria.testapp;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.google.firebase.FirebaseApp;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        FirebaseApp.initializeApp(this);
        setContentView(R.layout.activity_main);

        Button ospatarBTN = (Button) findViewById(R.id.ospatarBTN);
        Button tureBTN = (Button) findViewById(R.id.tureBTN);

        ospatarBTN.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, OspatarTask.class));
            }
        });

    }
}

File: RestaurantAdmin/app/src/main/java/com/gr232/restaurantadmin/MainActivity.kt
package com.gr232.restaurantadmin

import android.graphics.drawable.AnimationDrawable
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val animationDrawable : AnimationDrawable = loginActivityLayout.background as AnimationDrawable
        animationDrawable.setEnterFadeDuration(3500)
        animationDrawable.setExitFadeDuration(3500)
        animationDrawable.start()
    }
}